README.md

```diff
@@ -145,6 +145,16 @@ This image is configured with volumes at `/data` and `/datalog` to hold the Zook
 
 > Be careful where you put the transaction log. A dedicated transaction log device is key to consistent good performance. Putting the log on a busy device will adversely affect performance.
 
+## How to configure logging
+
+This image is configured with volume at `/logs` to hold logging files. By default, ZooKeeper redirects stdout/stderr outputs to the console (zkServer start-foreground) or file `/logs/zookeeper*.out` (zkServer start), and Log4j redirects to stdout/stderr. To separate Log4j logging files, just replace the redirection to `CONSOLE` with `ROLLINGFILE`, by passing environment variable `ZOO_LOG4J_PROP` as follows:
```

Comment:
I think this paragraph contains to much details, which makes it a bit unclear. How about making it more concise? Something like
> By default, ZooKeeper redirects stdout/stderr outputs to the console. You can redirect to a file located in `/logs` by passing environment variable `ZOO_LOG4J_PROP` as follows...

> This image is configured with a volume at `/logs` for your convenience.

Reply:
Thanks a lot! It's constructive and very helpful.